### ISSUE TITLE
Deploy alpha release from monorepo

### ISSUE THREAD

**Issue by Braden-Preston** [HarvestHaven/TPOT-Next#70]
Preserve the existing app in a separate package. Change netlify.toml so that it deploys from /packages/legacy properly.

At some point the beta branch would be merged into develop to become the new develop branch once the Beta Prototype milestone is complete and tested.
